src/common/base/CuckooHashCache.h

```diff
@@ -0,0 +1,875 @@
+#pragma once
```

Comment:
`#pragma once` is not standard,  please use include guard as other headers in our project.